### PR TITLE
Fix code climate SSL certificate error

### PR DESCRIPTION
**Pull request by nunorafaelrocha**

### CODE REVIEW

**Comment by nunofgs** [comment]
It shouldn't be necessary to manually post the data according to https://github.com/codeclimate/php-test-reporter/issues/3#issuecomment-96829384. Can you investigate?

**Comment by nunorafaelrocha** [comment]
According with [Jonathan Powers on 10 August](https://github.com/codeclimate/php-test-reporter/issues/3), the SSL error still persists in some projects and they were not able to fully resolve the issue.

So we will maintain the workaround for necessary time until codeclimate fully fix the issue.

File: .travis.yml [hunk 1]
```diff
@@ -12,10 +12,11 @@ before_script:
   - composer self-update
   - composer update --prefer-source
 
-script: phpunit
+script: vendor/bin/phpunit
```
Comment (ruimarinho) [inline .travis.yml]: Doesn't composer create symlinks for binaries?

Comment (nunorafaelrocha) [inline .travis.yml]: Travis comes with a pre-installed version of phpunit. We want to ensure that all our tests uses the version that is on the composer.json

**Pull request merged**